docs/release-notes/RELEASE-NOTES-0.12.0.md

```diff
@@ -1,22 +1,20 @@
 # About this release
 
-[This release][this-release] is focused on TODOFILLMEIN.
+[This release][this-release] is focused on faster sync - approx. 10 times faster.
 It:
-* Regulate the maximum number of concurrent connections (`sync` > `max_connections`) and the number of processes accepting connections (`sync` > `acceptors`).
-* Expose more configuration previously hard-coded, (`sync` > `connect_timeout`), (`sync` > `first_ping_timeout`) and (`sync` > `noise_hs_timeout`).
-* Improve the chain representation by keeping track of fork points. It reduces the need for block-by-block traversals, which in turn speeds up many chain operations. This impacts the persisted DB.
 * Changes the sync strategy, making it possible to more efficiently sync a chain from multiple nodes.
```

Comment:
Be more explicit about what has changed. E.g., 
* Changes the sync strategy, making it possible to sync a chain much more efficiently by focusing the sync on syncing with individual forks instead of simply syncing with individual peers.

Reply:
Addressed in new commit. cc @hanssv